AsyncDisplayKit/Details/_ASDisplayViewAccessiblity.mm

```diff
@@ -112,11 +159,24 @@ - (id)accessibilityElementAtIndex:(NSInteger)index
     return nil;
   }
   
-  return _accessibleElements[index];
+  ASDisplayNode *viewNode = self.asyncdisplaykit_node;
+  if (viewNode == nil) {
+    return nil;
+  }
+
+  // Recalculate the elements
+  NSArray *elements = SortAccessibilityElements(CollectAccessibilityElementsForViewAndNode(self, viewNode));
+
+  // We have to update the frame of the accessiblity element in case it changed
```

Comment:
This solution seems fine.  Just add something like: // the accessibilityFrame is in screen coordinates, so a scroll view moving is a common case where this value needs to change without this element able to detect this (besides performing the coordinate conversation again)